### DISCUSSION TITLE
Allow for non local IP connections

### DISCUSSION THREAD

**Discussion by PhillipDiCarlo** [Kounex/obs_station#19] [Ideas]
My biggest use for an app like this would be to connect to my computer when doing an IRL stream to control scenes and stop/start streaming. However, as the app stands, you can only enter local IP addresses instead of global IP address/dynamicIP service (ie duckDNS) and port forward. Would really love to see this simple addition added.

**Comment by Kounex** [comment]
Is part of the new 3.0 update - give it a try! :)

**Reply by SammiTea** [reply]
Hi, I saw the option to put in a Domain, but I did not get it running. Is there any good (video) tutorial explaining how to open the ports correctly on windows 11? I tried for hours but don't get it running when not connected to local network. I already put firewall (Defender) exeptions in and out and on the router forwarded the port.
Thanks

**Reply by PhillipDiCarlo** [reply]
Hello, I had this same issue with a different app. You want to hit "Windows Key" and search for firewall. Then from there add OBS.exe to the allowed programs list. You should be able to find a tutorial by searching "Allow program through firewall windows 11"
